src/web/components/Branding.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { LinkStyle } from '@root/src/amp/components/elements/TextBlockComponent';
```

Comment:
I could be wrong but I think we have a bit of an invisible rule where we don't use amp components in DCR?

Reply:
Maybe we could use `Link` from `src-link`? If you look in discussion you'll see how we combine `Link` with a wrapper to get pillar styles. Not sure if @SiAdcock  has introduced a `PillarLink version since then?

Reply:
Yeh, we maybe want to lint against that

Reply:
Ah yes. Good spot. Was from the initial copy I took of the AMP branding component.